src/Javinizer/Private/Convert-JVTitle.ps1

```diff
@@ -81,21 +81,26 @@ function Convert-JVTitle {
                 try {
                     $id = ($file | Select-String $RegexString).Matches.Groups[$RegexIdMatch].Value
                     $partNum = ($file | Select-String $RegexString).Matches.Groups[$RegexPtMatch].Value
+
+                    # If ID#### and there's no hypen, subsequent searches will fail
+                    if($id -match '^([a-z]+)(\d+)$') {
+                        $id = $Matches[1] + "-" + ($Matches[2] -replace '^0{1,5}', '').PadLeft(3, '0')
+                    }
                 } catch {
                     Write-JVLog -Write:$script:JVLogWrite -LogPath $script:JVLogPath -WriteLevel $script:JVLogWriteLevel -Level Debug -Message "File [$file] not matched by regex"
                     break
                 }
                 if ($fileBaseNameUpper -eq 1) {
                     if ($partNum -ne '') {
-                        $fileBaseNameUpper = "$id-pt$PartNum"
+                        $fileBaseNameUpper = "$id-PT$PartNum"

```

Comment:
I'm not sure about the stylistic change of making the `pt` uppercase. I think it just feels cleaner when it's lowercase in contrast to the movie ID. 

`ABP-420-PT1.mp4` vs `ABP-420-pt1.mp4`
or 
`118ABP00420-PT1.mp4` vs `118ABP00420-pt1.mp4`

Reply:
I believe that the final written output will still be in lowercase, as this was hardcoded in `Convert-JVString`,

```powershell
if ($PartNumber) {
   $convertedName += "-pt$PartNumber"
}
```

I have also tested it and the final output is in lowercase. This is of course when `$Rename` is set to true, otherwise, I think no renaming should happen.

The reason why I made it uppercase was for consistency with the name variable `$fileBaseNameUpper` which was converted to all uppercase for various string manipulation/checks in the `Convert-JVTitle`.

Actually, almost all checks are done with regex and as it appears it's case insensitive by default, so the changes were merely for consistency with the variable name `$fileBaseNameUpper` which expects all characters to be in uppercase. I guess this is just a stylistic preference, but I could revert it back if you prefer.

Reply:
Ah, good catch. See.. even I forgot the full functionality of that function. That should be fine then.